docs/push.md

```diff
@@ -0,0 +1,37 @@
+# Push Notifications
+
+**Note: Enterprise customers-only**
+
+**Note: Beta-only**
+
+We support Push Notifications for both Android (using [Firebase](https://firebase.google.com/docs/notifications/?hl=es)) and iOS (using [APN](https://developer.apple.com/notifications/)) devices. In order to enable and use Push Notifications, the following steps are required:
+
+- Register (only once) in SkedGo backend the corresponding credentials/certificates for each platform.
```

Comment:
How would an external user that? By emailing us or can they do it in the 3scale interface?